src/components/routes/Host/ListingForm/AsideContent.tsx

```diff
@@ -0,0 +1,213 @@
+import * as React from 'react';
+import ListItem from 'shared/ListItem';
+import Svg from 'shared/Svg';
+import LazyImage from 'shared/LazyImage';
+
+interface AsideContentInterface {
+  [name: string]: React.ReactNode;
+}
+
+const HomeType = () => (
+  <div>
+    <h3>Type of Home</h3>
+    <br />
+    <h4><strong>Entire place</strong></h4>
+    <p>The whole space is dedicated to guests. This may be a house, apartment, condo, loft, etc.</p>
+    <br />
+    <h4><strong>Private room</strong></h4>
+    <p>A single room within a place is dedicated to guests. While other areas could be shared, guests have their own private room for sleeping.</p>
+    <br />
+    <h4><strong>Shared room</strong></h4>
+    <p>Bedrooms or a common area that could be shared with others are dedicated to guests.</p>
+  </div>
+);
+
+const Title = () => (
+  <div>
+    <h3>Listing Name</h3>
+    <br />
+    <p>&quot;Beautiful Home with Wifi and Views&quot;</p>
+  </div>
+);
+
+const Description = () => (
+  <div>
+    <h3>Listing Description</h3>
+    <br />
+    <p>Include a summary of your place to help guests imagine what it will be like to stay there. Note what makes your place special, mention what the neighborhood is like, and add anything else guests should know before arriving.</p>
+  </div>
+)
+
+const FullAddress = () => (
+  <div>
+    <h3>Full Address</h3>
+    <br />
+    <p>When adding the Full Address -- don’t forget to add a zip code!</p>
+  </div>
+);
+
+const ListingPicUrl = () => (
+  <div>
+    <h3>Cover Photo</h3>
+    <br />
+    <p>High-quality photos are a must as our guests are typically business travelers. Ideal image dimensions are 1440x960. The maximum file size is 5 MB and the file types we accept are .jpg &amp; .png.</p>
+    <br />
+    <h4>Examples:</h4>
+    <br />
+    <div className="images-container">
+      <LazyImage className="image-container--horizontal" src="https://s3-us-west-2.amazonaws.com/beenest-public/images/photo-examples/Do_LA_Bedroom.jpg" />
```

Comment:
small comment but can we use the `https://static.beenest.com` url for these?

Reply:
👍